packages/reporter/src/commands/command.tsx

```diff
@@ -24,12 +24,14 @@ import HiddenIcon from '@packages/frontend-shared/src/assets/icons/general-eye-c
 import PinIcon from '@packages/frontend-shared/src/assets/icons/object-pin_x16.svg'
 import RunningIcon from '@packages/frontend-shared/src/assets/icons/status-running_x16.svg'
 
-const md = new Markdown({ breaks: true })
-
 const displayName = (model: CommandModel) => model.displayName || model.name
 const nameClassName = (name: string) => name.replace(/(\s+)/g, '-')
 
-export const formattedMessage = (message: string) => {
+export const formattedMessage = (message: string, type: string) => {
+  // we don't want <br> in our error messages, but allow it in Cypress.log
+  const mdObj = type === 'error' ? {} : { breaks: true }
+  const md = new Markdown(mdObj)
```

Comment:
I worry there's a performance cost to instantiating this every time we format a message. An alternative would be to instantiate one with `breaks: true` and one without and then choose which one to use when formatting messages here.

Reply:
True story. I'll refactor this.

Reply:
https://github.com/cypress-io/cypress/pull/28496/commits/a2c8ca322523d858a5ba69fe69d2a89e4ad58871